packages/@romejs/js-compiler/api/analyzeDependencies.test.ts

```diff
@@ -27,208 +28,313 @@ async function testAnalyzeDeps(input: string, sourceType: ConstSourceType) {
   'discovers import star',
   async (t) => {
-    await t.snapshot(await testAnalyzeDeps(`
-      import * as bar from 'foobar';
-    `, 'module'));
+    await t.snapshot(
+      await testAnalyzeDeps(
+        dedent`
+          import * as bar from 'foobar';
+        `,
+        'module',
+      ),
+    );
   },
 );
 
 test(
   'discovers import specifiers',
   async (t) => {
-    await t.snapshot(await testAnalyzeDeps(`
-      import {bar, foo, default as lol, ya as to} from 'foobar';
-    `, 'module'));
+    await t.snapshot(
+      await testAnalyzeDeps(
+        dedent`
+          import {bar, foo, default as lol, ya as to} from 'foobar';
+        `,
+        'module',
+      ),
+    );
   },
 );
 
 test(
   'discovers import default',
   async (t) => {
-    await t.snapshot(await testAnalyzeDeps(`
-      import bar from 'foobar';
-    `, 'module'));
+    await t.snapshot(
+      await testAnalyzeDeps(
+        dedent`
+          import bar from 'foobar';
+        `,
+        'module',
+      ),
+    );
   },
 );
 
 test(
   'discovers commonjs exports',
   async (t) => {
-    await t.snapshot(await testAnalyzeDeps(`
-      exports.yes = function() {};
-    `, 'script'));
+    await t.snapshot(
+      await testAnalyzeDeps(
+        dedent`
+          exports.yes = function() {};
+        `,
+        'script',
+      ),
+    );
   },
 );
 
 test(
   'discovers commonjs module.exports',
   async (t) => {
-    await t.snapshot(await testAnalyzeDeps(`
-      module.exports = function() {};
-    `, 'script'));
+    await t.snapshot(
+      await testAnalyzeDeps(
+        dedent`
+          module.exports = function() {};
+        `,
+        'script',
+      ),
+    );
   },
 );
 
 test(
   'discovers top level await',
   async (t) => {
-    await t.snapshot(await testAnalyzeDeps(`
-      await foobar();
-    `, 'module'));
+    await t.snapshot(
+      await testAnalyzeDeps(
+        dedent`
+          await foobar();
+        `,
+        'module',
+      ),
+    );
   },
 );
 
 test(
   'correctly identifies a file with es imports as es',
   async (t) => {
-    await t.snapshot(await testAnalyzeDeps(`
-      import 'bar';
-    `, 'module'));
+    await t.snapshot(
+      await testAnalyzeDeps(
+        dedent`
+          import 'bar';
+        `,
+        'module',
+      ),
+    );
   },
 );
 
 test(
   'correctly identifies a file with es exports as es',
   async (t) => {
-    await t.snapshot(await testAnalyzeDeps(`
-      export const foo = 'bar';
-    `, 'module'));
+    await t.snapshot(
+      await testAnalyzeDeps(
+        dedent`
+          export const foo = 'bar';
+        `,
+        'module',
+      ),
+    );
   },
 );
 
 test(
   'correctly identifies a file with cjs exports as cjs',
   async (t) => {
-    await t.snapshot(await testAnalyzeDeps(`
-      exports.foo = 'bar';
-    `, 'module'));
+    await t.snapshot(
+      await testAnalyzeDeps(
+        dedent`
+          exports.foo = 'bar';
+        `,
+        'module',
```

Comment:
I believe this test is wrong and this line should be changed to:
```suggestion
        'script',
```